firecracker/generate_firecracker_kernel.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+################################################################################
+# Usage: Call this script from the Makefile, which prevents unnecessarily
+#   recompiling the kernel (takes 4-5m on our Chromebooks).
+################################################################################
+readonly FIRECRACKER_RELEASE="v1.0.0"
+readonly KERNEL_CONFIG="resources/guest_configs/microvm-kernel-x86_64-4.14.config"
+
+REPOSITORY_ROOT=$(git rev-parse --show-toplevel)
+readonly REPOSITORY_ROOT
+
+readonly BUILD_DIR="/tmp/firecracker_kernel_build"
```

Comment:
Any reason to not use `mktemp` here?

Reply:
There is, actually! If you find yourself doing subsequent builds, the git clone already exists in that previous location. 

In fact, if I removed/changed
```
rm -rf build/kernel
```
you'd keep some object files too

small gains, but yeah

Reply:
but eh i'll mktemp it